src/Driver/SwiftDriver.php

```diff
@@ -16,6 +16,9 @@ class SwiftDriver implements DriverInterface
 
     public function __construct(array $settings, $transport = null)
     {
+        if (!array_key_exists('username', $settings) && !array_key_exists('password', $settings)) {
```

Comment:
Can you add this check on the `setUsername` / `setPassword` calls below? Also would like to see a test case added for a missing username and password.